airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -43,11 +42,6 @@
     ['conn_id', 'account_name', 'share_name', 'mount_path', 'read_only'],
 )
 
-DEFAULT_ENVIRONMENT_VARIABLES = {}  # type: Dict[str, str]
```

Comment:
Any reason to remove these (including the DEFAULT_MEMORY_IN_GB) instead of inlining as done in the constructor? It reads better with constants, and I believe that was a change made in the original PR

Reply:
We could also use `or` at the constructor level.

Should we try to conform with the style of the [kubernetespodoperator](https://github.com/apache/airflow/blob/master/airflow/contrib/operators/kubernetes_pod_operator.py) ?

Reply:
Re-implemented.